pythonforandroid/recipes/coincurve/__init__.py

```diff
@@ -0,0 +1,32 @@
+import os
+from pythonforandroid.recipe import PythonRecipe
+
+
+class CoincurveRecipe(PythonRecipe):
+    version = '18.0.0'
+    url = 'https://github.com/ofek/coincurve/archive/v{version}.tar.gz'
+    call_hostpython_via_targetpython = False
+    depends = [('python2', 'python3crystax'), 'setuptools',
```

Comment:
Hi @T-Dynamos !

`python2` and `python3crystax` are not in our recipe list anymore.

Can you please update the recipe accordingly?